docs/guides/cloud/branch-review.mdx

```diff
@@ -0,0 +1,178 @@
+- How Branch Review works and what might impact available data
+
+:::
+
+:::info
+
+&#8239;<Icon name="exclamation-triangle" /> Branch Review is currently in beta.
+See [how to enable](/guides/cloud/branch-review#Public-Beta-Access) this feature below.
+
+:::
+
+## Overview
+
+Cypress Branch Review is designed to elevate your pull request review workflow. It allows you to quickly identify the impact a pull request might have on your test suite in a single view. Compare which tests are failing, flaky, pending, added, or modified between the source and base branches and prevent the merging of low-quality code.
+
+A common scenario throughout the software development lifecycle (SDLC) is an engineer's **feature** branch that will be merged into the repo's **develop** or **main** branch.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/branch-review.jpg"
+  alt="Cypress Branch Review"
+/>
+
+Previously, pinpointing changes in your test suite's results required a manual side-by-side comparison between your newly-introduced branch's test runs and your base branch's test runs. This sub-optimal workflow often fails to answer the fundamental questions, _what changed_ and _why?_ You might be left wondering if the same tests are flaky between branches, when new failures were introduced, or if you added sufficient test coverage to your new branch.
+
+:::info
+
+Cypress Branch Review is currently only available for [GitHub integrated projects](/guides/cloud/integrations/source-control/github). We are working on adding support for GitLab and Bitbucket soon.
+
+:::
+
+## Public Beta Access
+
+:::tip
+
+To access the Branch Review public beta, an admin can simply enable it under "Cypress Labs" in your Organization Settings in Cypress Cloud.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/labs-beta.png"
+  alt="Enable Branch Review Beta"
+  width="90%"
+/>
+
+:::
+
+## Getting Started
+
+Branch Review works by leveraging the Cypress Cloud GitHub integration to query the GitHub API for branches with pull requests. Your project will first need to be connected to a GitHub repository. If you haven't already, follow the steps in [Install the Cypress GitHub app](/guides/cloud/integrations/source-control/github#Install-the-Cypress-GitHub-app) to connect your project to GitHub.
+
+### Accessing Branch Review
+
+To access Branch Review, navigate to the **Branches** tab in the left sidebar and select the branch you want to review. In order to see a comparison, the branch you select must have a pull request.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/get-to-branch-review.jpg"
+  alt="Get to Branch Review"
+  caption="Active branches, shown with available PR # tags"
+/>
+
+If you do not see PR # tags associated with a branch, you may need to [pass PR numbers](/guides/continuous-integration/introduction#CI-Build-Information) to runs as environment variables. This helps ensure Cypress Cloud can map PR data correctly.
```

Comment:
Grammar edit: should be "run", not "runs"

Reply:
> Ah, I see what you mean, but I think it's a matter of context. Here we're saying you'll pass the PR # as an option to your runs. As in they get "passed to the runs", on the command line, for example. Not that they get passed to "run as environment variables". Does that make sense?

Reply:
got a 👍 in slack